20-minimal/README.md

```diff
@@ -6,7 +6,6 @@ with the full-sized s2i-enabled Node.JS 20 image to build the application. The i
 but compared to the full-sized Node.JS 20 image it will be missing many build-time dependencies.
 Users can choose between RHEL, CentOS and Fedora based images.
 The RHEL images are available in the [Red Hat Container Catalog](https://access.redhat.com/containers/),
-the CentOS images are available on [Quay.io](https://quay.io/organization/centos7),
```

Comment:
What about CentOS Stream 9 images? I guess it should be replaced by https://quay.io/organization/sclorg, right?

Reply:
Done.